src/99-appendices/08-coordinate-systems.md

```diff
@@ -116,47 +116,35 @@ below, the origin is at the AC and the orientation of the axes is RAS. Unless
 
-Hybrid spaces combining surface and volume data used with CIFTI files.
+### Nonstandard template identifiers
 
-| Coordinate System | Description                                                                                                                                                                                                                                                                                                                                                                                                           |
-| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| HCPMNIfsLR32k     | Combination of `MNI152NLin6AsymConte69` subcortical volumetric space (with 2mm grid) and `fsLR32k` surface space resulting in 91282 data points per timepoint. Used in Human Connectom Project. [https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/91282_Greyordinates](https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/91282_Greyordinates)      |
-| HCPMNIfsLR59k     | Combination of `MNI152NLin6AsymConte69` subcortical volumetric space (with 1.6mm grid) and `fsLR59k` surface space resulting in 170494 data points per timepoint. Used in Human Connectom Project. [https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/170494_Greyordinates](https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/170494_Greyordinates) |
+| Coordinate System | Description                                                                                                                                                                                                                                                           |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| individual        | Participant specific anatomical space (for example derived from T1w and/or T2w images). This coordinate system requires specifying an additional, participant-specific file to be fully defined. In context of surfaces this space has been refered to as `fsnative`. |
+| custom            | Custom space defined using a group/study-specific template. This coordinate system requires specifying an additional file to be fully defined.                                                                                                                        |
```

Comment:
So I've conceded on the controlled vocabulary issue, but I would say if that's the case, we need to have more than just `custom`, as you'll want to distinguish between some custom cases. Here we identify a specific use case: study-specific templates. Why not add `study`?

Are there any others that are common that are worth carving out?

Reply:
This is a great point. I'll add `study` and try to think of other possibilities.